lib/app/routes/comments.rb

```diff
@@ -10,6 +10,9 @@ class Comments < Core
           redirect "/"
         end
         comment = CreatesComment.create(submission.id, current_user, params[:body])
+
+        comment.increment_five_a_day if comment.qualifying?
```

Comment:
I would say `user.increment_five_a_day` rather than `comment.increment_five_a_day`. I really like the `comment.qualifying?` bit. It's very readable!